docs: Add router example docs to networking section (#855)
acorn-io/acorn: docs/docs/50-running/02-networking.md

If you set your cluster domain to `my-company.com`, deployed into the default `acorn` namespace, named your app `blog`, and the container name in the Acornfile was `default`, the published endpoint for your app would be: `http://blog.my-company.com`

In addition to this method of controlling your endpoint, you can publish to an explicit external name using the `--publish` flag. See the [publishing ports](#publish-individual-ports) section for more details.

> ## Routers

Reviewer: Just a nit-pick, but could we add newlines to match the rest of this document?
```suggestion

## Routers

```

> If you have two containers normally they would be exposed as two different HTTP services. The router feature allows you to expose these two containers as a single HTTP service with separate routes. Let's take a look at how we can achieve this below.

Reviewer: What are your thoughts on this? Personally I think that it makes it clearer that this scales to more than two containers which may be non-obvious for beginners.

```suggestion
If you have multiple containers normally they would be exposed as two different HTTP services. The router feature allows you to expose those containers as a single HTTP service with separate routes. Let's take a look at how we can achieve this with two containers below.
```

Author: I like this better

Starting with a sample Acornfile that exposes two services

> ```acorn

Reviewer: Follow up from the previous nit-pick, could we try to make the formatting with newlines match the rest of the document? Applies to the rest of this document.

```suggestion

```acorn

```

Author: fixed
